packages/media-utils/src/use-windowed-audio-data.ts

```diff
@@ -96,11 +103,9 @@ export const useWindowedAudioData = ({
 
 			signal.addEventListener('abort', cont, {once: true});
 
-			const source = new UrlSource(src);
-
-			using input = new Input({
+			const input = new Input({
```

Comment:
Now we are not disposing the input anymore, it is being kept indefinitely.

While this makes sense in preview, won't this make it more likely for us to hit the Chrome connection limit during render?

We had this problem in the retreat, won't we just reintroduce it again?

Maybe it makes sense to differentiate between renders and preview as well!